NuRadioMC/simulation/simulation2.py

```diff
@@ -16,6 +16,7 @@
 import NuRadioReco.modules.io.eventWriter
```

Comment:
It might be a little late for this, but how do people feel about renaming simulation2.py to something else in the py3 release? It has been rather sad, that our main file is called soemthing like work in progress.

Reply:
yes we should to that! Now is the right time